remote-build-coordinator/src/main/java/org/jboss/pnc/remotecoordinator/maintenance/DefaultRemoteBuildsCleaner.java

```diff
@@ -173,24 +172,35 @@ private Result deleteBuildsFromIndy(BuildRecord buildRecord, String authToken) {
                     }
                 }
             } catch (IndyClientException e) {
-                String description = MessageFormat
-                        .format("Error in deleting generic http repos for build {0}: {1}", buildContentId, e);
```

Comment:
This seems intentional here to have also the exception message in the log in case it is displayed without stacktrace, although not sure if this log is read and displayed anywhere nowadays.

Reply:
Slf4j loggers display stack traces if an Exception is last argument in **error** and **warn** levels. The previous way would make it display twice.

Reply:
Yeah, the SLF4J implementation might have changed since this code was written. Let's leave it like this and in case it causes some unclear logging, we can put it back. But it seems it shouldn't.